lib/Db/Mailbox.php

```diff
@@ -0,0 +1,76 @@
+<?php declare(strict_types=1);
```

Comment:
newline or?

Reply:
Saw @icewind1991 using this style and I kinda like it. objections?

Reply:
Ah no fine by me. Just didn't see it like this before ;)